packages/web-react/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -36,24 +31,17 @@ export const Breadcrumbs = <T extends ElementType = 'nav'>(props: SpiritBreadcru
           items?.map((item, index) => (
             <React.Fragment key={`BreadcrumbsItem_${item.title}`}>
               {index === items.length - 2 && goBackTitle && (
-                <li className={displayTabletNoneClassName}>
-                  <Icon name="chevron-left" />
-                  <Link href={item.url} color="primary" isUnderlined>
-                    {goBackTitle}
-                  </Link>
-                </li>
+                <BreadcrumbsItem href={item.url} iconStart={<Icon name="chevron-left" />} isGoBackOnly>
```

Comment:
I would avoid the iconStart prop. We don't need it, do we? It wasn't editable before.

Reply:
and iconEnd

Reply:
Please suggest a better solution. I think it is a matter of time before someone would like to change the chevron icon to whatever else.

Of course, I can hardcode the logic of this into the BreadcrumbsItem component but it seems to me to not be the best way.

Reply:
I suggest we discuss this in person and in a bigger group :)

Reply:
Refactored to `iconNameStart` and `iconNameEnd` props.